flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/BaseExpressions.java

```diff
@@ -1407,6 +1409,44 @@ public OutType arrayUnion(InType array) {
                 unresolvedCall(ARRAY_UNION, toExpr(), objectToExpression(array)));
     }
 
+    /**
+     * Returns an array of the elements in the concat of array1 and array2, without duplicates.
```

Comment:
Does this comment still match the method ?
I'm asking since it is unclear: what is `array1`, `array2` and why we are talking about `without duplicates`?

Reply:
fixed it. Thank you!